Source/DataSources/ModelVisualizer.js

```diff
@@ -150,6 +154,43 @@ define([
     };
 
     /**
+     * Computes a bounding sphere which encloses the visualization produced for the specified entity.
+     *
+     * @param {Entity} entity The entity whose bounding sphere to compute.
+     * @param {BoundingSphere} result The bounding sphere onto which to store the result.
+     * @returns {BoundingSphereState} BoundingSphereState.DONE if the result contains the bounding sphere,
+     *                       BoundingSphereState.PENDING if the result is still being computed, or
+     *                       BoundingSphereState.FAILED if the entity has no visualization in the current scene.
+     */
+    ModelVisualizer.prototype.getBoundingSphere = function(entity, result) {
+        //>>includeStart('debug', pragmas.debug);
+        if (!defined(entity)) {
+            throw new DeveloperError('entity is required.');
+        }
+        if (!defined(result)) {
+            throw new DeveloperError('result is required.');
+        }
+        //>>includeEnd('debug');
+
+        var modelData = this._modelHash[entity.id];
+        if (!defined(modelData)) {
+            return BoundingSphereState.FAILED;
+        }
+
+        var model = modelData.modelPrimitive;
+        if (!defined(model) || !model.show) {
+            return BoundingSphereState.FAILED;
+        }
+
+        if (!model.ready) {
+            return BoundingSphereState.PENDING;
+        }
+
+        BoundingSphere.transform(model.boundingSphere, model.modelMatrix, result);
```

Comment:
Actually, that doesn't work because the z up to y up transformation is already baked into that computed matrix and the scale is already baked into `model.boundingSphere`.  The end result is a bounding sphere that is incorrectly centered and way bigger than it needs to be.

I just checked the doc, and it says `model.scale` is not baked into the `model.boundingSphere`, but it actually is.  Should we update the doc, or the code?

In order to transform the bounding sphere with the current code, I had to divide radius by the scale to get back the original value, then transform the resulting bounding sphere and then re-apply the new scale.  Easy to do once I had it and everything works as expected.

```
var boundingSphere = model.boundingSphere;
result.radius = boundingSphere.radius / model.scale;
result.center = Cartesian3.multiplyByScalar(boundingSphere.center, model.scale, result.center);
BoundingSphere.transform(result, model.modelMatrix, result);
result.radius *= model.scale;
```